ricecooker/utils/carousel.py

```diff
@@ -0,0 +1,174 @@
+from bs4 import BeautifulSoup, NavigableString
+from bs4.element import Tag
+from urllib.parse import urlsplit
+from itertools import zip_longest
+import requests
+import hashlib
+import shutil
+import os
+from ricecooker.utils import add_file
+
+html = """
```

Comment:
What's the best way of referring to this HTML file's location from inside code? I can't work out if/where pip is installing it to, I can't see it in my site-packages.

Reply:
Ivan's taken me through how to do this, but since there's work on a new solution for image libraries, I think it's not worth adding this to the codebase in hindsight. Removed carousel.py.